docs/core/windows-prerequisites.md

```diff
@@ -45,8 +45,8 @@ See [.NET Core 1.x Supported OS Versions](https://github.com/dotnet/core/blob/ma
 * Installing .NET Core via a *.zip* file. This can include build/CI/CD servers.
 
 > [!NOTE]
-> *For Windows 7 and Windows Server 2008 machines only:*
-> Make sure that your Windows installation is up-to-date and includes hotfix [KB2533623](https://support.microsoft.com/help/2533623) installed through Windows Update.
+> *For Windows versions below Windows 10:*
+> Make sure that your Windows installation is up-to-date and includes [KB2999226](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows) which can be installed through Windows Update. If you do not have this, you will see an error when you launch a .NET Core application like this `The program can't start because api-ms-win-crt-runtime-1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem.`.
```

Comment:
do not have this -> don't have this update installed

you will -> you'll 

like this -> like the following: